zookeeper-server/src/main/java/org/apache/zookeeper/ClientCnxnSocketNetty.java

```diff
@@ -18,47 +18,45 @@
 
 package org.apache.zookeeper;
 
+import io.netty.bootstrap.Bootstrap;
```

Comment:
nit: this import order is different from the one on master branch.

Reply:
Done.